Formula/d/difftastic.rb

```diff
@@ -4,6 +4,7 @@ class Difftastic < Formula
   url "https://github.com/Wilfred/difftastic/archive/refs/tags/0.60.0.tar.gz"
   sha256 "4346b269240f3beff153758d0a2313accc79645ee21b36cfcc71ac0fca5d9b5d"
   license "MIT"
+  revision 1
```

Comment:
no need to do the revision

```suggestion
```

Reply:
Okay, I wasn’t sure. Thanks for the clarification.